signer/Cargo.toml

```diff
@@ -15,7 +15,7 @@ description = "Sign extrinsics to be submitted by Subxt"
 keywords = ["parity", "subxt", "extrinsic", "signer"]
 
 [features]
-default = ["sr25519", "ecdsa", "subxt", "std", "native"]
+default = ["sr25519", "ecdsa", "eth", "subxt", "std", "native"]
```

Comment:
I wonder whether this should be a default feature because I reckon "eth accounts/signer" is rather niche use case and most users won't need it.

Doesn't matter that much it's easy to opt-out anyway, @jsdw thoughts?

Reply:
The only reason I tend to default to having all feature flags is for nicer IDE/docs support (otherwise loads of stuff is greyed out). So this is consistent with what I did with the other signers, but I'd be very open to figuring out an approach which has IDEs/docs working nicely even without feature flags enabled here; I just haven't bothered :)